pkg/services/store/entity/sqlstash/data/entity_history.sql

```diff
@@ -0,0 +1,30 @@
+SELECT {{ template "common_entity_select_into" . }}
+
+    FROM {{ .Ident "entity_history" }} AS e
+
+    WHERE 1 = 1
+
+        {{ if gt .Before 0 }}
+            AND {{ .Ident "resource_version" }} < {{ .Arg .Before }}
+        {{ end }}
+
+        {{/* There are two mutually exclusive search modes: by GUID and by Key */}}
+
+        {{ if ne .Query.GUID "" }}
+            AND {{ .Ident "guid" }} = {{ .Arg .Query.GUID }}
+
+        {{ else }}
+            AND {{ .Ident "group" }}    = {{ .Arg .Query.Key.Group }}
+            AND {{ .Ident "resource" }} = {{ .Arg .Query.Key.Resource }}
+            AND {{ .Ident "name" }}     = {{ .Arg .Query.Key.Name }}
+
+            {{ if ne .Query.Key.Namespace "" }}
+                AND {{ .Ident "namespace" }} = {{ .Arg .Query.Key.Namespace }}
+            {{ end }}
```

Comment:
Do we need this in the If statement ?

Reply:
Yes, it is possible to query for entities on all namespaces.